src/App.vue

```diff
@@ -120,13 +193,20 @@
       previewInput() {
         if (this.previewInputValid) {
           // Copy data over
+          JSON.stringify(this.previewInput)
```

Comment:
This line doesn't appear to do anything (the result isn't assigned to a variable).